lib/styles/styles.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class Styles {
+  // For light theme
+  static final Color eurekaBackgroundColor = Color(0xFF37474F);
+  // For dark theme
+  static final Color darkBackgroundColor = Color(0xFF256580);
+  static final Color eurekaAppBarColor = Color(0xFF402580);
+
+  Styles._();
+
+  static final ThemeData darkTheme = ThemeData(
+    visualDensity: VisualDensity.adaptivePlatformDensity,
+    scaffoldBackgroundColor: darkBackgroundColor,
+    primaryColor: eurekaAppBarColor,
```

Comment:
This solution seems like it's a good way to do it, but you need to provide it for all items. This includes wherever a color is hard coded in the app. There should not be any hardcoded colors left in any components or screen. 

Example of why this is needed: 
- When using dark mode on the phone, the side menu is still white -  ![image](https://user-images.githubusercontent.com/36316432/111856465-233e2e80-88e8-11eb-8f99-4eb84d6dc561.png)
- Or When using a pure black background, the text isn't readable - 
![image](https://user-images.githubusercontent.com/36316432/111856493-5e406200-88e8-11eb-9dda-76160d31bdd5.png)

Reply:
Mentioned in what I did: Switching to dark mode was something I implemented on the fly. I did not include a way to change the colors for the buttons because we will not have a scafoldbackground that is pure black. The background colors and parts of the the app bar will change color when dark mode is on. Since we do not have a set dark mode we agreed on this functionality and colors were not implemented.

Reply:
@tonynguyen98 For the side menu due to how the component was created. There is not way to change the background color of the side menu.